### ISSUE TITLE
Galaxy install warnings

### ISSUE THREAD

**Issue by mdklatt** [mdklatt/idea-ansible-plugin#3]
Using `ansible-galaxy` results in a warning when installing to a local project directory, even though the install location matches the [Ansible collections documentation](https://docs.ansible.com/ansible/latest/collections_guide/collections_installing.html#installing-collections-with-ansible-galaxy), and the project playbooks can successfully use the collection. This seems to be a nuisance warning, so resolving it might be outside the scope of this project.

The final message, "Skipping install, no requirements found", is from `ansible-galaxy role install` because there are no roles specified in the requirements file. Ideally, the Galaxy run configuration should only execute `collection install` and `role install` if necessary, but this will require interpreting the requirements file.

```
sh -c ".venv/bin/ansible-galaxy collection install -r ansible/requirements.yml -p ansible/collections && .venv/bin/ansible-galaxy role install -r ansible/requirements.yml -p ansible/roles"
[WARNING]: The specified collections path
'/home/touserproject/ansible/collections' is not part of
the configured Ansible collections paths
'/home/user/.ansible/collections:/usr/share/ansible/collections'. The
installed collection won't be picked up in an Ansible run.
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Downloading https://galaxy.ansible.com/download/community-docker-2.7.1.tar.gz to /home/user/.ansible/tmp/ansible-local-5718_0l49arc/tmp9y8mu2ux/community-docker-2.7.1-eisjxjnf
Installing 'community.docker:2.7.1' to '/home/user/project/ansible/collections/ansible_collections/community/docker'
community.docker:2.7.1 was installed successfully
Skipping install, no requirements found
```

**Comment by mdklatt** [comment]
Commit 12685095b0a37bf4466dadf73e0b99c9f09bced5 adds a basic requirements file parser and only calls `ansible-galaxy roles` or `ansible-galaxy collections` if required.

**Comment by mdklatt** [comment]
Commit be11a8e9c4d524e6cf5aa8002f9ee01aa78d0903 sets `ANSIBLE_ROLES_PATH` and `ANSIBLE_COLLECTIONS_PATH` for `ansible-galaxy` execution, which silences warnings about unconfigured paths.

**Comment by mdklatt** [comment]
Issue resolved by commit be11a8e9c4d524e6cf5aa8002f9ee01aa78d0903.
